docs/cxx/cxx.rst

```diff
@@ -550,6 +578,52 @@ to allow constructors to "fail" gracefully.  Accordingly, a class like
 but requires a call to ``set_children()`` after construction.  Neither the
 Python API (nor the Swift API) would be subject to this limitation.
 
+The OpenTime and OpenTimelineIO libraries both have their own error
+definitions. The tables below outline the errors, which python exceptions they
+raise, and what their semantic meaning is.
+
+.. csv-table:: OpenTime Errors
+    :header: "Value", "Python Exception Type", "Meaning"
+    
+    OK, n/a, No Error
+    INVALID_TIMECODE_RATE, ``ValueError``, "Timecode rate isn't a valid SMPTE rate"
+    NON_DROPFRAME_RATE,  ``ValueError``, "Timecode rate isn't valid for SMPTE Drop-Frame Timecode"
+    INVALID_TIMECODE_STRING,  ``ValueError``, "String is not properly formatted SMPTE timecode string"
+    TIMECODE_RATE_MISMATCH,  ``ValueError``, " Timecode string has a frame number higher than the frame rate"
+    INVALID_TIME_STRING,  ``ValueError``,
+    NEGATIVE_VALUE,  ``ValueError``,
+    INVALID_RATE_FOR_DROP_FRAME_TIMECODE,  ``ValueError``, "Timecode rate isn't valid for SMPTE non Drop-Frame Timecode"
```

Comment:
```suggestion
    INVALID_RATE_FOR_DROP_FRAME_TIMECODE,  ``ValueError``, "Timecode rate isn't valid for SMPTE Drop-Frame Timecode"
```

Also, I think `INVALID_RATE_FOR_DROP_FRAME_TIMECODE` and `NON_DROPFRAME_RATE` are basically the same. Perhaps independent from this change, we could consolidate them.

Reply:
Good suggestion, filed issue #625 to track this.